### PR TITLE
Update Installation.md

### PR DESCRIPTION
**Pull request by gagb**
## Why are these changes needed?

Replace autogen->pyautogen in env setup to avoid confusion

Added deactivation example

## Related issue number

Accommodates suggestion in #211

## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [x] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

File: website/docs/Installation.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ When not using a docker container, we recommend using a virtual environment to i
 
 You can create a virtual environment with `venv` as below:
 ```bash
-python3 -m venv autogen
+python3 -m venv pyautogen
 source autogen/bin/activate
```
Comment (pcdeadeasy) [inline website/docs/Installation.md:12]: you can mention the user can execute deactivate to get out of the python environment.

File: website/docs/Installation.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ source autogen/bin/activate
 Another option is with `Conda`, Conda works better at solving dependency conflicts than pip. You can install it by following [this doc](https://docs.conda.io/projects/conda/en/stable/user-guide/install/index.html),
 and then create a virtual environment as below:
 ```bash
-conda create -n autogen python=3.10  # python 3.10 is recommended as it's stable and not too old
-conda activate autogen
+conda create -n pyautogen python=3.10  # python 3.10 is recommended as it's stable and not too old
+conda activate pyautogen
```
Comment (pcdeadeasy) [inline website/docs/Installation.md:21]: add conda deactivate as well to deactivate the environment

**Review by pcdeadeasy — CHANGES_REQUESTED** [review]

**Review by AaronWard — APPROVED** [review]
LGTM 👍

**Review by pcdeadeasy — APPROVED** [review]
LGTM

File: website/docs/Installation.md [hunk 3]
```diff
@@ -8,17 +8,27 @@ When not using a docker container, we recommend using a virtual environment to i
 
 You can create a virtual environment with `venv` as below:
 ```bash
-python3 -m venv autogen
+python3 -m venv pyautogen
 source autogen/bin/activate
```
Comment (sonichi) [inline website/docs/Installation.md:12]:
```suggestion
source pyautogen/bin/activate
```

**Review by sonichi — COMMENTED** [review]
One change before merging

**Comment by gagb** [comment]
> One change before merging

Good catch! Thanks Chi!

**Pull request merged**